metamod/src/game_support.cpp

```diff
@@ -10,21 +10,97 @@ const game_modinfo_t g_known_games[] = {
+	{ "stargatetc",         "stc_i386_opt.so",          "hl.dll",                    "StargateTC (Updated, 2.x)"                        },
+	{ "svencoop",           "hl.so",                    "hl.dll",                    "Sven Coop (Old)"                                  },
+	{ "svencoop",           "server.so",                "server.dll",                "Sven Coop (Steam)"                                },
+	{ "swarm",              "swarm.so",                 "swarm.dll",                 "Swarm"                                            },
+	{ "tfc",                "tfc.so",                   "tfc.dll",                   "Team Fortress Classic"                            },
+	{ "thewastes",          "thewastes.so",             "thewastes.dll",             "The Wastes"                                       },
+	{ "timeless",           "pt.so",                    "timeless.dll",              "Project Timeless"                                 },
+	{ "tod",                "hl.so",                    "hl.dll",                    "Tour of Duty"                                     },
+	{ "trainhunters",       "th.so",                    "th.dll",                    "Train Hunters"                                    },
+	{ "ts",	                "ts_i686.so",               "mp.dll",		         	 "The Specialists"                                  },
+	{ "ts",                 "ts_i386.so",               "mp.dll",                    "The Specialists"                                  },
+	{ "tt",                 "tt.so",                    "tt.dll",                    "The Trenches"                                     },
+	{ "underworld",         "uw.so",                    "uw.dll",                    "Underworld Bloodline"                             },
+	{ "valve",              "hl.so",                    "hl.dll",                    "Half-Life"                                        },
+	{ "vs",                 "vs.so",                    "mp.dll",                    "VampireSlayer"                                    },
+	{ "wantedhl",           "hl.so",                    "wanted.dll",                "Wanted!"                                          },
+	{ "wizardwars",         "wizardwars_i486.so",       "wizardwars.dll",            "Wizard Wars (Steam)"                              },
+	{ "wizardwars_beta",    "wizardwars_i486.so",       "wizardwars.dll",            "Wizard Wars Beta (Steam)"                         },
+	{ "wizwars",            "mp.so",                    "hl.dll",                    "Wizard Wars (Old)"                                },
+	{ "wormshl",            "wormshl_i586.so",          "wormshl.dll",               "WormsHL (Old)"                                    },
+	{ "wormshl",            "wormshl_i686.so",          "wormshl.dll",               "WormsHL (Steam)"                                  },
```

Comment:
There is already a code automatically lookup for all postfixes for linux versions.
See https://github.com/theAsmodai/metamod-r/blob/2c41e0ae1d629220794e30ba2c89c4a0076bdb95/metamod/src/utils.cpp#L3

Reply:
ok. and how it will be palced?  "wormshl_i586.so" -> "wormshl"?

Reply:
@stamepicmorg `wormshl.so`

Reply:
i am not shure but updated it at commit https://github.com/theAsmodai/metamod-r/pull/25/commits/7a0a9953b3ed579b82ac4fcfb9ca86ee96cb4d78

check plz?